include/SubPanel/SubPanelDefinitions.php

```diff
@@ -767,7 +767,7 @@ public function exclude_tab($tab_name)
      * return all available subpanels that belong to the list of tab modules.  You can optionally return all
      * available subpanels, and also optionally group by module (prepends the key with the bean class name).
      */
-    public function get_all_subpanels($return_tab_modules_only = true, $group_by_module = false)
+    public static function get_all_subpanels($return_tab_modules_only = true, $group_by_module = false)
```

Comment:
Would we need this to be static? the only place it seems to be called this way is in configuretabs.